app/models/CompanyInformation.scala

```diff
@@ -18,7 +18,7 @@ package models
 
 import play.api.libs.json.{Json, OFormat}
 
-case class CompanyInformation(name: String, address: AddressInformation)
+case class CompanyInformation(name: String, consent: String, address: AddressInformation)
```

Comment:
Should consent not be optional?

Reply:
change made in customs data store to default consent none to "0" for false.